zdm/data/MCMC/params.json

```diff
@@ -0,0 +1,101 @@
+{
+    "cube": {
+        "parameter_order": ["lC", "sfr_n", "alpha", "lmean", "lsigma", "lEmax", "gamma", "H0"]
+    },
+    "mcmc": {
+        "parameter_order": ["sfr_n", "alpha", "lmean", "lsigma", "lEmax", "lEmin", "gamma", "H0"]
+    },
+    "mcmc_full": {
+        "parameter_order": ["logF", "sfr_n", "alpha", "lmean", "lsigma", "lEmax", "lEmin", "gamma", "H0", "DMhalo", "lRmin", "lRmax", "Rgamma", "sigmaDMG"]
+    },
+    "config": {
+        "luminosity_function": 2,
+        "alpha_method": 0
+    },
+    "logF": {
+        "DC": "cosmo",
+        "min": -2.0,
+        "max": 0.0
+    },
+    "lEmax": {
+        "DC": "energy",
+        "min": 40.5,
+        "max": 45.0, 
+        "n": 10
+    },
+    "lEmin": {
+        "DC": "energy",
+        "min": 36.0, 

```

Comment:
the paper says 38.0 - 39.9 but this looks more consistent with Figure 1

Reply:
Oh nice catch here, I forgot to update that table...